tombo/tombo_text_output.xml

```diff
@@ -1,5 +1,5 @@
-<tool id="tombo_text_output" name="Tombo create" version="@VERSION@">
-    <description>text output</description>
+<tool id="tombo_text_output" name="Tombo text_output" version="@VERSION@">
+    <description>browser </description>
```

Comment:
That looks to technical. Can we leave the old one and add browser.

Reply:
text_output is the name of the command. how about 'browser output' or 'create text output'?